nexus/db-model/src/network_interface.rs

```diff
@@ -17,6 +17,13 @@ use nexus_types::identity::Resource;
 use omicron_common::api::external;
 use uuid::Uuid;
 
+/// The max number of interfaces that may be associated with a resource,
+/// e.g., instance or service.
+///
+/// RFD 135 caps instances at 8 interfaces and we use the same limit for
+/// all types of interfaces for simplicity.
+pub const MAX_NICS: usize = 8;
```

Comment:
There exists the same constant in Nexus itself, `crate::nexus::MAX_NICS_PER_INSTANCE`. I think this was moved, not new, but just checking we want to duplicate the const rather than re-export one.

Reply:
By usage they definitely have to be the same value, so it doesn't seem good to have two constants. I'll rename this one `MAX_NICS_PER_INSTANCE`, and change nexus to re-export it instead of defining its own. Nice catch.